storage/csi-cloning.md

```diff
@@ -0,0 +1,112 @@
+---
+title: CSI Cloning
+authors:
+  - "@chuffman"
+reviewers:
+  - "@gnufied”
+  - “@jsafrane”
+approvers:
+  - TBD
+  - "@..."
+creation-date: 2019-09-05
+last-updated: 2019-09-25
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# csi-cloning
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs] 
+
+## Summary
+
+We want to include the upstream CSI Cloning feature in OpenShift. 
+
+## Motivation
+
+* This has been requested by both the OCS and CNV teams
+* This feature is a core functionality offered by storage vendors that we need to enable for our customers
+
+### Goals
+
+* Backport the cloning features in the upstream external-provisioner.
```

Comment:
English question: should it be `from the upstream..` instead of `in the upstream`?

Reply:
Noticed in snapshot OEP: I think we don't need to backport anything, we just pick upstream 1.16  provisioner and release it downstream.

Reply:
Talked to @bertinatto, what we want is "rebase downstream external-provisioner to the upstream one released for Kubernetes 1.16"

Reply:
I've updated this text to indicate that we're rebasing the downstream external-provisioner.